apps/web/src/views/Notifications/containers/NotificationView.tsx

```diff
@@ -0,0 +1,197 @@
+      return unFilteredNotifications.filter((notification: NotifyClientTypes.NotifyMessageRecord) => {
+        const extractedType = notification.message.type
+        return extractedType === subscriptionType
+      })
+    }
+    const sortNotifications = (unFilteredNotifications: NotifyClientTypes.NotifyMessageRecord[]): any[] => {
+      switch (notificationType) {
+        case SubsctiptionType.Lottery:
+          return typeFilter(SubsctiptionType.Lottery, unFilteredNotifications)
+        case SubsctiptionType.Prediction:
+          return typeFilter(SubsctiptionType.Prediction, unFilteredNotifications)
+        case SubsctiptionType.Liquidity:
+          return typeFilter(SubsctiptionType.Liquidity, unFilteredNotifications)
+        case SubsctiptionType.Farms:
+          return typeFilter(SubsctiptionType.Farms, unFilteredNotifications)
+        case SubsctiptionType.PriceUpdates:
+          return typeFilter(SubsctiptionType.PriceUpdates, unFilteredNotifications)
+        case SubsctiptionType.Promotional:
+          return typeFilter(SubsctiptionType.Promotional, unFilteredNotifications)
+        case SubsctiptionType.Alerts:
+          return typeFilter(SubsctiptionType.Alerts, unFilteredNotifications)
+        default:
+          return notifications
+      }
+    }
+    return sortNotifications(notifications)
+  }, [notifications, notificationType])
+
+  return (
+    <Box width="100%">
+      <NotificationHeader
+        leftIcon={
+          <IconButton tabIndex={-1} variant="text" onClick={onDismiss}>
+            <ModalCloseButton onDismiss={onDismiss} />
+          </IconButton>
+        }
+        rightIcon={
+          <IconButton tabIndex={-1} variant="text" onClick={toggleSettings}>
+            <CogIcon color="primary" />
+          </IconButton>
+        }
+        text={t('Notifications')}
+      />
+      <Flex alignItems="center" justifyContent="space-between" paddingBottom="8px" paddingTop="4px" paddingX="22px">
+        <Box width="125px">
+          <Select onOptionChange={handleNotifyOptionChange} options={NotificationFilterTypes} />
+        </Box>
+        <NotificationsTabButton activeIndex={viewMode} setActiveIndex={setViewMode} />
+      </Flex>
+      {viewMode === ViewMode.Archived ? (
+        <>
+          <Flex justifyContent="center" pt="12px" pb="16px" borderBottom="1px solid" borderBottomColor="cardBorder">
+            <Text color="textSubtle">{t('All archived messages will be deleted in 24hr')}</Text>
+          </Flex>
+          <Flex justifyContent="center" pt="16px">
+            <Text color="textDisabled">{t('ARCHIVED MESSAGES')}</Text>
+          </Flex>
+        </>
+      ) : null}
+
+      <NotificationContainerStyled maxHeight="400px">
+        {viewMode === ViewMode.Latest ? (
+          <NotificationItem notifications={filteredNotifications} isClosing={isClosing} />
+        ) : (
+          <NotificationItem notifications={archivedNotifications} isClosing={isClosing} />
+        )}
```

Comment:
Can u check it this works?
`<NotificationItem notifications={viewMode === ViewMode.Latest ?  filteredNotifications  : archivedNotifications} isClosing={isClosing} />`

Reply:
yep it does. looks better too